.build/ci/generate-test-report.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh -e
```

Comment:
This script is a pure `ant` target, so I think we can move it under the `.build` directory and rename to `run-generate-test-report.sh`.

Reply:
it's only used by CI (it depends on structuring of files that is currently happening in `.jenkins/Jenkinsfile` 
…?

Reply:
We can move the whole `ci' folder under `.jenkins', as nobody uses it except Jenkins itself.

Reply:
they are intended to be standards (and/or helper scripts) for all CI systems.

`.build/ci/generate-test-report.sh` is to aggregate all test xml files.  it is intended to be used all all splits on a given test type/target/

`.build/ci/generate-ci_summary.sh` is to create the ci_summary.html sharable artefact. it expects a folder hierarchy that separates test types/targets.  (it will locally link each failure and testsuite to the junit html file found in this hierarchy, as it was generated by the above script.)

Adding this information in as header comments to each file